Source/IGListCollectionView.m

```diff
@@ -11,25 +11,6 @@
 
 @implementation IGListCollectionView
 
-- (void)commonInit {
-    self.backgroundColor = [UIColor whiteColor];
-    self.alwaysBounceVertical = YES;
-}
-
-- (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionViewLayout *)layout {
-    if (self = [super initWithFrame:frame collectionViewLayout:layout]) {
-        [self commonInit];
```

Comment:
@jessesquires ya and w/out the compiler we will likely break some stuff w/out warning. @dcaunt let's update and maybe open a new issue for that?

I'd love to consider removing the dependence on `IGListCollectionView` but admit I **love** the compiler warnings for those methods.